.travis.yml

```diff
@@ -29,3 +29,4 @@ script: ./tool/travis.sh
 cache:
   directories:
   - $HOME/.pub-cache
+  - $HOME/packages/devtools/.dart_tool/build
```

Comment:
Are we leaking something about how travis works with this `$HOME`? Packages in the build repo don't reference that variable...

Reply:
oh whoops! I totally just goofed here will fix